README.md

```diff
@@ -55,7 +60,7 @@ fn main() {
 }
 ```
 
-Alternativly you can see the example `basic.rs` located in the examples folder.
+Alternativly you can see the example `basic.rs` or `scroll.rs` located in the examples folder.
```

Comment:
Alternatively

Reply:
Got my first mechanical keyboard this week and still adjusting to it. However, I've now installed a spell checker extension so it shouldn't happen again, thanks for the feedback 😊